ColorPatterns/Scenes/Game/GameViewModel.swift

```diff
@@ -43,18 +52,18 @@ extension GameViewModel {
     }
 }
 
-private extension GameViewModel {
+private extension GameViewModelImpl {
 
     private func calculateScore() {
         if gameStarted {
             userScore.updateScorePoints(gameEngine.colorRange.contains(gameEngine.pickedColor))
-            delegate?.didUpdateScore(score: userScore.score)
+            controller?.updateScoreLabel(score: userScore.score)
         } else {
             gameEngine.userColor = gameEngine.pickedColor
         }
     }
 
-    //TODO: Remove prints when not needed
+    // TODO: Remove prints when not needed
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/67f2732f59af0d268971dc68cf5f930b621eb1ad/ColorPatterns/Scenes/Game/GameViewModel.swift#L66"></span>TODOs should be resolved (Remove prints when not needed).<br />
<code>todo</code> <code>GameViewModel.swift:66</code></td>
    </tr>
  </tbody>
</table>